var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -257,14 +264,18 @@ class Mapl(CMakePackage):
     depends_on("esmf~debug", when="~debug")
     depends_on("esmf+debug", when="+debug")
 
-    depends_on("gftl@1.13.0:", when="@2.45:")
+    depends_on("udunits", when="@2.48:")
+
+    depends_on("gftl@1.14.0:", when="@2.48:")
+    depends_on("gftl@1.13.0:", when="@2.45:2.47")
     depends_on("gftl@1.11.0:", when="@2.44")
     depends_on("gftl@1.10.0:", when="@2.40:2.43")
     depends_on("gftl@1.5.5:1.9", when="@:2.39")
 
     # There was an interface change in gftl-shared, so we need to control versions
     # MAPL 2.39 and older can use up to 1.6.0 but MAPL 2.40+ needs 1.6.1 or higher
-    depends_on("gftl-shared@1.8.0:", when="@2.45:")
+    depends_on("gftl-shared@1.8.0:", when="@2.48:")
```

Comment:
Is this a typo? If 2.45:2.47 depend on 1.8.0 and 2.48 as well, these lines can be combined

Reply:
Nuts. Yeah bad copy paste!